catalog/spatial/geocoding/pom.xml

```diff
@@ -33,5 +33,6 @@
         <module>spatial-geocoding-localsearch</module>
         <module>spatial-geocoding-websearch</module>
         <module>spatial-geocoding-admin-module</module>
+        <module>spatial-clavin-lucene</module>
```

Comment:
`spatial-geocoding-clavin`?  I feel like we should keep the naming convention the same in the geocoding module.  Maybe this would fit into the `spatial-geocoding-lucene` package?